common/persistence/serialization/task_serializer.go

```diff
@@ -34,28 +34,18 @@ import (
 
 	enumsspb "go.temporal.io/server/api/enums/v1"
 	persistencespb "go.temporal.io/server/api/persistence/v1"
+	"go.temporal.io/server/common/definition"
 	"go.temporal.io/server/common/primitives/timestamp"
 	"go.temporal.io/server/common/tasks"
 )
 
 type (
 	TaskSerializer struct {
```

Comment:
should this be combined with serialization.Serializer?

Reply:
sure, will do in this PR